test/specs/annotation/tests/03.table_display.js

```diff
@@ -62,6 +63,9 @@ exports.execute = function (options) {
         var tableCatalogAnnotEntityUri = options.url + "/catalog/" + catalog_id + "/entity/" + schemaName + ":" +
             tableNameCatalogAnnot;
 
+        var tableCompactOptionsEntityUri = options.url + "/catalog/" + catalog_id + "/entity/" + schemaName + ":" +
```

Comment:
Change `tableNameCatalogAnnot` to`tableNameCompactOptions`.

Reply:
I didn't notice test cases are failing, so I approved it. But now I realized why your new test case is failing. You just need to change the table name to be the new table that you have created.

Reply:
Ah, thank you. I couldn't run the tests locally last night so I opened the PR so the tests could run and I would debug them after. Thank you.